### ISSUE TITLE
Skip .bat files

### ISSUE THREAD

**Issue by mcandre** [duggan/shlint#16]
When I run shlint over my `scripts/` folder, it erroneously reports:

```
$ shlint scripts/
checkbashisms -n -p bin/*
script scripts/ehrlich.bat does not appear to have a #! interpreter line;
```

Please filter out any MS-DOS Batch (.bat/.BAT) files passed into shlint or checkbashisms.
